cmake/GenerateHelptags.cmake.in

```diff
@@ -1,10 +1,10 @@
 if(DEFINED ENV{DESTDIR})
   file(TO_CMAKE_PATH
-    "$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/share/nvim/runtime/doc"
+    "$ENV{DESTDIR}/@CMAKE_INSTALL_FULL_DATADIR@/nvim/runtime/doc"
```

Comment:
Bah.  The problem here is that we have to use `install(SCRIPT ...)` and that doesn't allow for arguments. :-(  So there really is little choice in the matter.